### PR TITLE
Update Russian Translation to 2020 Regulation

### PR DESCRIPTION
**Pull request by TimWCA**
Update Russian Translations of the WCA Regulations and Guidelines to Version 2020-01-01.

### CODE REVIEW

**Comment by Ivan-Brigidano** [comment]
@TimWCA is the translation finished?

**Comment by TimWCA** [comment]
Yes

С уважением,
Тимофей Вихрянов

________________________________

File: russian/wca-guidelines.md [hunk 1]
```diff
@@ -71,7 +73,7 @@
 - 3h++) [ПРИМЕР] Примеры совершенствования основной структуры: возможны новые движения, невозможны обычные движения, видно больше элементов или граней, видны цвета на задней стороне, движения выполняются автоматически, больше или другие собранные состояния.
 - 3h2++) [УТОЧНЕНИЕ] Головоломки "из цветного пластика", которые существенно отличаются от большинства головоломок "из цветного пластика" серийного производства, можно использовать только по решению Делегата WCA.
 - 3j+) [НАПОМИНАНИЕ] Раньше было разрешено использовать головоломки с элементами, имеющими гравировки или рельефы. Теперь это запрещено.
-- 3j++) [УТОЧНЕНИЕ] В Clock'е кнопки не должны быть отличимы от других кнопок на той же стороне.
+- 3j++) [УТОЧНЕНИЕ] В Clock'е кнопки не должны могут отличаться от других кнопок на той же стороне.
```
Comment (Niksson) [inline russian/wca-guidelines.md]: I think there is a typo, did you mean this:

```suggestion
- 3j++) [УТОЧНЕНИЕ] В Clock'е кнопки могут отличаться от других кнопок на той же стороне.
```

Comment (TimWCA) [reply russian/wca-guidelines.md]: Thanks! I will fix it.

Comment (TimWCA) [reply russian/wca-guidelines.md]: Thanks! I will fix it.

File: russian/wca-regulations.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-<version>Версия: 1 мая 2019 г.
+<version>Версия: 1 января 2020 г.
 
 
 ## Замечания
 
 ### Перевод
 
-Это не официальные Положения WCA, а вольный перевод. В случае возникновения сомнений следует обращаться к оригинальным Положениям. В переводе возможны ошибки! Вопросы и замечания по поводу перевода отправляйте по электронной почте на [nick.masson@yandex.ru](mailto:nick.masson@yandex.ru).
+Это не официальные Положения WCA, а вольный перевод. В случае возникновения сомнений следует обращаться к оригинальным Положениям. В переводе возможны ошибки! Вопросы и замечания по поводу перевода отправляйте по электронной почте на [vihrianov1@outlook.com](mailto:vihrianov1@outlook.com).
```
Comment (Niksson) [inline russian/wca-regulations.md]: I think you shouldn't leave your own e-mail for contacts (at least for now). Let's leave it as it is now

Comment (TimWCA) [reply russian/wca-regulations.md]: Ok.

File: russian/wca-regulations.md [hunk 3]
```diff
@@ -100,6 +100,7 @@
     - 2k3) Ведет себя незаконным или неприличным образом или умышленно наносит ущерб предметам или личному имуществу в месте проведения соревнования.
     - 2k4) Мешает или отвлекает других во время соревнования.
     - 2k5) Не соблюдает Положения WCA во время соревнования.
+    - 2k6) A1c) Не удовлетворяет требованиям дисциплины (например, должен знать, как решать головоломку). Участник не должен участвовать, если ожидается, что у него будет результат DNF или намеренно плохой. Наказание: дисквалификация попытки (DNF) или дисквалификация с дисциплины (см. [Положение 2j](regulations:regulation:2j)), по решению Делегата WCA.
```
Comment (Niksson) [inline russian/wca-regulations.md]: A1c reference should be removed:
```suggestion
    - 2k6) Не удовлетворяет требованиям дисциплины (например, должен знать, как решать головоломку). Участник не должен участвовать, если ожидается, что у него будет результат DNF или намеренно плохой. Наказание: дисквалификация попытки (DNF) или дисквалификация с дисциплины (см. [Положение 2j](regulations:regulation:2j)), по решению Делегата WCA.
```

File: russian/wca-regulations.md [hunk 4]
```diff
@@ -203,30 +205,30 @@
...
     - 7h1) Команда организаторов может обязать участника, которого вызвали соревноваться, оставаться в обозначенной зоне для участников до окончания всех его попыток в данном раунде.
 
 
 ## <article-9><events><events> Статья 9: Дисциплины
 
-- 9a) WCA проводит соревнования по головоломкам, известным как "twisty puzzles", сборка которых производится вращением граней.
+- 9a) WCA проводит соревнования по механическим головоломкам, известным как "twisty puzzles", сборка которых производится вращением граней.
 - 9b) Официальные головоломки и форматы дисциплин WCA:
-    - 9b1) Кубик 3x3x3, Кубик 2х2х2, Кубик 4х4х4, Кубик 5х5х5, Clock, Мегаминкс, Пирамидка, Square-1, Скьюб, сборка кубика 3x3x3 одной рукой и сборка кубика 3x3x3 ногами.
+    - 9b1) Кубик 3x3x3, Кубик 2х2х2, Кубик 4х4х4, Кубик 5х5х5, Clock, сбокра кубика 3х3х3 одной рукой, Мегаминкс, Пираминкс, Скьюб и Square-1.
```
Comment (Niksson) [inline russian/wca-regulations.md]: There is a typo here:
```suggestion
    - 9b1) Кубик 3x3x3, Кубик 2х2х2, Кубик 4х4х4, Кубик 5х5х5, Clock, сборка кубика 3х3х3 одной рукой, Мегаминкс, Пираминкс, Скьюб и Square-1.
```

File: russian/wca-regulations.md [hunk 5]
```diff
@@ -464,25 +467,14 @@
...
-    - D3b) Участник убирает ступни с таймера, чтобы начать сборку.
-- D4) Окончание сборки:
-    - D4a) Участник ставит ступни на сенсоры таймера, останавливая таймер.
-
-
 ## <article-E><fewest-moves><fewestmovessolving> Статья E: Сборка на количество ходов
 
 - E2) Процедура сборки на количество ходов:
-    - E2a) Судья раздаёт скрамблы и бумагу всем участникам, затем запускает секундомер и объявляет начало попытки.
+    - E2a) Судья раздаёт скрамблы и бумагу всем участникам, затем запускает секундомер и объявляет начало попытки. Судья запускает секундомер и говорит "Начали".
```
Comment (Niksson) [inline russian/wca-regulations.md]: You are repeating yourself here, I think it should go like this:
```suggestion
    - E2a) Судья раздаёт скрамблы и бумагу всем участникам. Судья затем запускает секундомер и говорит "Начали".
```

Comment (TimWCA) [reply russian/wca-regulations.md]: Ok. Thanks!

File: russian/wca-regulations.md [hunk 6]
```diff
@@ -492,23 +484,27 @@
...
 
 ## <article-F><clock><clocksolving> Статья F: Сборка Clock
 
 - F1) Процедура такая же, как и в [статье A](regulations:article:A) (скоростная сборка). Ниже описаны дополнительные положения, которые замещают соответствующие положения в [статье A](regulations:article:A).
 - F2) Судья ставит перемешанную головоломку на мат в вертикальном положении.
+    - F2a) Команда организаторов может заставить использовать подставки для головоломки, чтобы она не падала до начала попытки. Если подставки применяются, об этом должно быть объявлено до начала раунда.
+        - F2a1) Судья должен убрать подставку с мата сразу после того, как участник убрал с неё головоломку.
 - F3) После преинспекции участник ставит головоломку в вертикальном положении. Штраф: дисквалификация попытки (DNF).
-    - F3a) Участнику запрещается менять положения кнопок до начала сборки. Штраф: дисквалификация попытки (DNF).
+	- F3a) Участнику запрещается менять положения кнопок до начала сборки. Штраф: дисквалификация попытки (DNF).
```
Comment (Niksson) [inline russian/wca-regulations.md]: There should be 1 tabulation here, isn't it?

**Comment by Niksson** [comment]
Also, about changing the Pyraminx name from "Пирамидка" to "Пираминкс". Historically, the former is somewhat a common nickname for the puzzle and we didn't change it, but the latter is a valid name too (a literal translation of the name). I think "Пираминкс" is fine, I don't think that people would not understand, what this puzzle is. And IIRC, @Claster mentioned that the site translation mentions "Пираминкс", so the new translation should be consistent with the site.

File: russian/wca-regulations.md [hunk 7]
```diff
@@ -203,30 +205,30 @@
         - 7f1c) Таймер: таймер стакмата или секундомер (для долгих попыток).
         - 7f1d) Поверхность: плоская поверхность, на которой располагается стакмат. Мат считается частью поверхности. Таймер не считается частью поверхности.
     - 7f2) Таймер должен быть прикреплен к мату и расположен на поверхности, располагаясь на ближней к участнику стороне мата.
-        - 7f2a) Исключение: для сборки 3x3x3 ногами стакмат с матом должен быть расположен прямо на полу. Таймер может быть расположен на дальней от участника стороне мата.
+    - 7f3) Команде организаторов следует модифицировать стакмат, чтобы избежать случайного нажатия кнопок (например, использовать кольца вокруг кнопок), а также защитить батарейки (например, заполнив батарейный отсек).
```
Comment (Niksson) [inline russian/wca-regulations.md]: I don't like the wording here. "To secure" here doesn't have the same meaning as "protect", it's rather "fix in place", and "padding the battery compartment" isn't the same as just "fill the battery compartment" (that's too vague). 
A pad is something soft to fill between the battery compartment and the battery itself. I suggest to reflect this in the translation with some concrete examples of what things you could use as padding:
```suggestion
    - 7f3) Команде организаторов следует модифицировать стакмат, чтобы избежать случайного нажатия кнопок (например, использовать резиновые кольца вокруг кнопок), а также более плотно прижать батарейку в отсеке (например, добавлением между батарейкой и крышкой кусочка бумаги или поролона).
```

**Comment by hanwu85** [comment]
Hello @TimWCA , @Niksson 
We will merge this one when you think the translations are ready,

**Comment by Niksson** [comment]
Hello @hanwu85! 

The translation is ready for merging. @TimWCA, thanks for helping out!

**Pull request merged**